guava/src/com/google/common/math/LongMath.java

```diff
@@ -805,7 +828,8 @@ public static long factorial(int n) {
    *
    * @throws IllegalArgumentException if {@code n < 0}, {@code k < 0}, or {@code k > n}
    */
-  public static long binomial(int n, int k) {
+  @SuppressWarnings("lowerbound:compound.assignment.type.incompatible")// the lowest n can be is 0 in this method
```

Comment:
This is a fact that's already conveyed by the `@NonNegative` annotation on `n`. You should explain *why* `n` cannot actually violate the invariant - the annotations already express what the invariant is.